models/comments-model.js

```diff
@@ -29,4 +29,34 @@ exports.selectCommentsByArticleId = (article_id) => {
       const comment = rows[0];
       return comment;
     });
+};
+
+exports.selectCommentById = (comment_id) => {
+  const queryString = 'SELECT * FROM comments WHERE comment_id = $1';
+
+  return db
+    .query(queryString, [comment_id])
+    .then(({ rows }) => {
+      const commentExists = rows[0];
+  if (!commentExists) {
+        return Promise.reject({
+          status: 404,
+          msg: 'comment does not exist',
+        });
+      }
+      return commentExists;
+    });
+};
+
+
+exports.deleteCommentById = (comment_id) => {
+  const queryString = 'DELETE FROM comments WHERE comment_id = $1';
+
+  return db
+    .query(queryString, [comment_id])
+    .then(({ rows }) => {
+      const deletedComment = rows[0];
+
+      return deletedComment;
+    });
```

Comment:
At this point you aren't returning anything from your delete statement - after a `DELETE` in PSQL, you need a `RETURNING *` to get information back that you have deleted.

This means what you're returning is actually just `undefined`

This isn't an issue, because you don't want to actually return your comment and aren't doing anything with it, and because you are checking the comment exists before deleting it, you know something will be deleted